sway/lock.c

```diff
@@ -48,6 +55,13 @@ static void handle_output_commit(struct wl_listener *listener, void *data) {
 
 static void handle_surface_destroy(struct wl_listener *listener, void *data) {
 	struct sway_session_lock_surface *surf = wl_container_of(listener, surf, destroy);
+
+	// Move the seat focus to another surface if one is available
```

Comment:
Hm. I'm not a fan of this hack.

Reply:
Replaced with tracking the focus in server state; that also simplifies the input changes and improves the stability of focus on output creation.